src/index.js

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Calculator from './components/Calculator';
+import './index.scss';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```

Comment:
- [ ] I would recommend not delete the `App` component because you want one parent component inside which you can put all your components to keep the flow easier.
- [ ] The `App.js` component should be class-based.
- [ ] The `App.js`(main component) should have the imported Calculator.js component.

Reply:
eslint's react/prefer-stateless-function has been disabled in App.js so it can be turned into a class, otherwise, eslint asks for it to be a function